wp-content/themes/core/styles/blocks/group/dark.json

```diff
@@ -6,25 +6,51 @@
 	"blockTypes": [ "core/group" ],
 	"styles": {
 		"color": {
-			"background": "var(--wp--preset--color--base-black)",
-			"text": "var(--wp--preset--color--base-white)"
+			"background": "var(--wp--preset--color--black)",
+			"text": "var(--wp--preset--color--white)"
 		},
 		"elements": {
 			"button": {
 				"color": {
-					"background": "var(--wp--preset--color--base-white)",
-					"text": "var(--wp--preset--color--base-black)"
+					"background": "var(--wp--preset--color--white)",
+					"text": "var(--wp--preset--color--black)"
 				},
 				":hover": {
 					"color": {
-						"background": "var(--color-neutral-20)",
-						"text": "var(--wp--preset--color--base-black)"
+						"background": "var(--color-blue)",
+						"text": "var(--wp--preset--color--white)"
 					}
 				},
 				":focus": {
 					"color": {
-						"background": "var(--color-neutral-20)",
-						"text": "var(--wp--preset--color--base-black)"
+						"background": "var(--color-dark-blue)",
+						"text": "var(--wp--preset--color--white)"
+					}
+				},
+				":active": {
+					"color": {
+						"background": "var(--color-dark-blue)",
+						"text": "var(--wp--preset--color--white)"
+					}
+				}
+			},
```

Comment:
❓ Just checking that these are the globally defined styles for buttons in the Moose style guide? In the dark theme, I don't see any blues at all for the buttons: http://p.tri.be/i/uTQNaS

Reply:
The prototype I was give had the hover in blue originally. Looks like that may have changed now and I'll update it to white with black text.